### DISCUSSION TITLE
Windows program to add text overlay on camera image

### DISCUSSION THREAD

**Discussion by eliangbb** [AxisCommunications/discussions#20] [ACAP] [answered]
Hi there, 
Is there a way to build Windows .exe program (in c/c++) and run .exe to add text overlay on the camera video by using Windows SDK or other SDKs? 
I built sample axoverlay APP (.eap file) by using ACAP native SDK, the APP can be added/installed/ran the on the camera, and the text overlay worked. I want to make this text overlay work from Windows .exe program instead of using APP. Is there a way to do so? 
Any help will be greatly appreciated!

**Comment by pataxis** [comment, marked as answer]
Since you have got overlay to work with an ACAP application (.eap-file) and you wonder if it's possible to do this in an Windows application instead I will close this discussion under `ACAP` category and let the discussion continue in #19 marked with category `Axis Windows SDK`.
